lib/chef/http/json_output.rb

```diff
@@ -61,6 +61,7 @@ def handle_response(http_response, rest_request, return_value)
           [http_response, rest_request, return_value]
         else
           Chef::Log.debug("Expected JSON response, but got content-type '#{http_response['content-type']}'")
+          Chef::Log.debug("Response body contains:\n#{http_response.body}")
```

Comment:
Can we do `http_response.body.length < 100 ? http_response.body : "#{http_response.body[0..100]}..."`?